zookeeper/core/component.py

```diff
@@ -368,6 +367,44 @@ def __component_init__(instance, **kwargs):
     ) | set(kwargs)
 
 
+####################################
+# Configuring the child components #
+###################################
+
+
+def __configure_children__(instance, conf):
+    for field_name, field_config in conf.items():
+        configure(getattr(instance, field_name), field_config)
+
+
+def __default_post_configure__(instance, conf):
+    print(conf)
+    instance.__configure_children__(conf)
+
+
+def _wrap_post_configure(component_cls: Type) -> None:
```

Comment:
Although the `-> None` return type annotation is correct, could we remove it here to be consistent with the other functions in this file?